german.html

```diff
@@ -33,92 +33,29 @@
     <div class='about'>
       <div class='row'>
         <div class='col-12 center'>
-          <h2>Dass Juniors awesome sind, weiss ich. Gern weiter ausarbeiten.</h2>
+          <h2>Die JAA Slack Kanal ist jetzt geschlossen</h2>
         </div>
       </div>
 
       <div class='row'>
         <div class='col-12'>
           <p>
-            Organisationen wie die Rails Girls helfen Anfänger*innen, das Programmieren für sich zu entdecken. Aber was tun, wenn du den nächsten Schritt wagen möchtest?
+            Seit kurzem haben wir die JAA Slack kanal geschlossen. Obwohl es uns sehr gefreut hat die Kanal zu leiten, es ist uns nötig Zeit an 
+            andere Ziele zu fokusieren.
```

Comment:
"Obwohl es uns sehr gefreut hat die Kanal zu leiten, es ist uns nötig Zeit an andere Ziele zu fokusieren." -->

"Obwohl es uns sehr gefreut hat den Kanal/Channel zu leiten, werden wir unsere Zeit nutzen, um uns auf andere Ziele zu fokussieren." (Sort of depends on what you wanna say)